renderjson.js

```diff
@@ -50,8 +50,14 @@
 //     .object.syntax ("{", "}")
 //     .array.syntax  ("[", "]")
 
-var module;
-(module||{}).exports = renderjson = (function() {
+(function (renderjson) {
+    if (typeof module === "object") {
+        module.exports = renderjson;
+    }
+    if (typeof window !== "undefined") {
+        window.renderjson = renderjson;
+    }
+}) (function() {
```

Comment:
Let's just do:

```javascript
(module||{}).exports = (window||{}).renderjson = (function() {
```
I'd like the boilerplate junk to take as little room as possible...